docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -115,7 +115,7 @@ class [[eosio::contract]] smrtcontract : public contract {
 
 The property used for the second parameter must be of template type `std::tuple`. The first parameter must be the type of the property indexed non-uniquely, in our case the type `std::string` is used because `first_name` is the property indexed non-uniquely. And the last parameter of the tuple type must be the type of a property name which is unique. In our case the type `eosio::name` is used because property `account_name` is unique. Multiple properties can be indexed non-uniquely as well. In this case the first parameters types correspond to the properties being indexed. And, as previously already mentioned, the last parameter correspond to the type of a property name which is unique.
```

Comment:
Should not start a sentence with and

Reply:
i.e.

The first parameter must be the type of the property indexed non-uniquely, in our case the type `std::string` is used because `first_name` is the property indexed non-uniquely. ***** And the last parameter of the tuple type must be the type of a property name which is unique.

Reply:
> Should not start a sentence with and
not according to this:
https://www.grammarly.com/blog/starting-a-sentence-with-a-conjunction/#:~:text=There%20is%20nothing%20wrong%20with,%2C%E2%80%9D%20or%20other%20similar%20conjunctions.&text=Some%20readers%20especially%20dislike%20seeing,words%20does%20sometimes%20seem%20melodramatic.